node/tests/ServerModules.test.ts

```diff
@@ -3237,66 +3349,5 @@ describe("Server Module Tests", () => {
                 newIndex,
             );
         });
-
-        it("FT._ALIASLIST test", async () => {
```

Comment:
please restore
I recommend not to move/rearrange tests, because it is easy to lose one